lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -656,7 +661,8 @@ def set_payload(record, entry)
       # available, or if the only remainaing key is 'message'.
       if @service_name == CLOUDFUNCTIONS_SERVICE && @cloudfunctions_log_match
         entry['textPayload'] = @cloudfunctions_log_match['text']
-      elsif @service_name == CLOUDFUNCTIONS_SERVICE && record.key?('log')
+      elsif (@service_name == CLOUDFUNCTIONS_SERVICE ||
+             @service_name == CONTAINER_SERVICE) && record.key?('log')
```

Comment:
might you want to do this only if 'log' is the sole remaining field (like 'message' below) to allow for the possibility of using struct if there are additional fields?